src/libsync/propagateuploadng.cpp

```diff
@@ -506,7 +506,7 @@ void PropagateUploadFileNG::startNextChunk()
     }
 
     QMap<QByteArray, QByteArray> headers;
-    // headers["OC-Chunk-Offset"] = QByteArray::number(_sent);
+    headers["OC-Chunk-Offset"] = QByteArray::number(_currentChunk);
```

Comment:
Incidentally `_currentChunk` -> `_currentChunkOffset`?

Reply:
Yes `_currentChunk` is now the offset. Hence all the changes to `quint64`.

Reply:
I'd rather you don't change the meaning of currentChunk for the normal (non zsync) upload.
This is going to cause problems, for example while resuming. (The logic in PropagateUploadFileNG::slotPropfindFinished will break)